### ISSUE TITLE
✔️ 🐞 Bug report: Warning: call_user_func_array() expects parameter 1 to be a valid callback, function 'wp_print_media_templates' not found or invalid function name

### ISSUE THREAD

**Issue by pratikrm** [elementor/elementor#18368]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

The code used [here](https://github.com/elementor/elementor/blob/master/core/app/modules/onboarding/module.php#L431), shows a PHP warning on Rank Math's Setup Wizard page. Adding a function_exists condition before using the `wp_print_media_templates` function will fix the issue:

```
add_action( 'admin_menu', function() {
    if ( function_exists( 'wp_print_media_templates' ) ) {
        add_action( 'wp_print_footer_scripts', 'wp_print_media_templates' );
    }
} );
```

### Steps to reproduce

1. Install [Rank Math](https://wordpress.org/plugins/seo-by-rank-math/) plugin
2. Navigate to Dashboard >> Rank Math >> Setup Wizard
3. See a warning in the footer

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [ ] I can reproduce this bug consistently using the steps above.

### System Info

					
== Server Environment ==
	Operating System: Darwin
	Software: nginx/1.16.0
	MySQL version: MySQL Community Server - GPL v8.0.16
	PHP Version: 7.4.1
	PHP Memory Limit: 256M
	PHP Max Input Vars: 4000
	PHP Max Post Size: 1000M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.9.3
	Site URL: http://rankmath.local
	Home URL: http://rankmath.local
	WP Multisite: No
	Max Upload Size: 300 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: 0
	Admin Email: dev-email@flywheel.local
	Debug Mode: Active

== Theme ==
	Name: Twenty Twenty-One
	Version: 1.5
	Author: the WordPress team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/99.0.4844.84 Safari/537.36

== Active Plugins ==
	Classic Editor
		Version: 1.6.2
		Author: WordPress Contributors

	Elementor
		Version: 3.6.2
		Author: Elementor.com

	Rank Math SEO
		Version: 1.0.86
		Author: Rank Math

	WooCommerce
		Version: 6.3.1
		Author: Automattic

== Must-Use Plugins ==
	BPS MU Tools
		Version: 7.0
		Author: AITpro

<br />
<b>Notice</b>:  Undefined index: title in <b>/Users/aayansh/Local Sites/rankmath/app/public/wp-content/plugins/elementor/core/experiments/experiments-reporter.php</b> on line <b>89</b><br />

== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Accessibility Improvements: Active by default
	Import Export Template Kit: Active by default
	Additional Custom Breakpoints: Active by default
	Hide native WordPress widgets from search results: Active by default
	: Inactive by default
	Flexbox Container: Inactive by default
	Landing Pages: Active by default
	Color Sampler: Active by default
	Favorite Widgets: Active by default
	Admin Top Bar: Active by default

== Log ==
	
Log: showing 13 of 132022-02-09 10:14:20 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.4.1',
  'to' => '3.6.0',
)]
2022-02-09 10:14:20 [info] elementor-pro::elementor_pro_updater Started 
2022-02-09 10:14:20 [info] Elementor Pro/Upgrades - _on_each_version Start  
2022-02-09 10:14:20 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2022-02-09 10:14:20 [info] Elementor Pro/Upgrades - _v_3_5_0_price_list Start  
2022-02-09 10:14:20 [info] Elementor Pro/Upgrades - _v_3_5_0_price_list Finished 
2022-02-09 10:14:20 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.4.1',
  'to' => '3.6.0',
)]
2022-02-09 10:14:21 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.4.1',
  'to' => '3.6.0',
)]
2022-04-06 06:50:14 [info] elementor::elementor_updater Started 
2022-04-06 06:50:14 [info] Elementor/Upgrades - _on_each_version Start  
2022-04-06 06:50:14 [info] Elementor/Upgrades - _on_each_version Finished 
2022-04-06 06:50:14 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.5.5',
  'to' => '3.6.2',
)]
2022-04-06 06:50:14 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.5.5',
  'to' => '3.6.2',
)]

PHP: showing 2 of 2PHP: 2022-02-21 04:00:32 [notice X 2][/Users/aayansh/Local Sites/rankmath/app/public/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-02-21 04:01:44 [notice X 1][/Users/aayansh/Local Sites/rankmath/app/public/wp-content/plugins/elementor/includes/template-library/classes/class-import-images.php::186] Undefined index: url [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

== Elementor - Compatibility Tag ==

**Comment by nivnoimanelementor** [comment]
@pratikrm We are looking into it, thanks

**Comment by toineenzo** [comment]
Looks like this got fixed in 3.10, awesome!

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this issue was fixed in Elementor **v3.10**! 🥳 

✔️ Please feel free to check it out and update your plugin to the new version!

Cheers 🥂
